site/assets/scss/_content.scss

```diff
@@ -2,13 +2,12 @@
 // Bootstrap docs content theming
 //
 
-.bd-content {
-  // Offset content from fixed navbar when jumping to headings
-  > :target {
-    padding-top: 5rem;
-    margin-top: -5rem;
-  }
+// Offset content from fixed navbar when jumping to headings
+:root {
+  scroll-padding-top: 5rem;
```

Comment:
If I remember well, the removal of this property was to avoid the unexpected scrolling while typing into the search bar. (#33993)

Reply:
Oh I see! I put this PR in Draft mode in order to find a better solution. Any help would be helpful if you have some ideas.

Reply:
With the new DocSearch v3 system, we don't have this issue anymore since there is a popup instead of a typeahead. It seems to work pretty well in desktop and mobile modes.

Reply:
Clicking the version dropdown picker also causes a scrolling issue—each time you click it, the page moves up a bit.

Reply:
Argh. OK I see it on mobile. Opening the offcanvas menu, each time I click outside of it makes the page moves up a bit.